puzzle-utils/words/dropquote.py

```diff
@@ -169,40 +179,31 @@ def snapshot(self):
                 word.possibilities))
         
         return snapshot
+    
+    def print(self):
+        print('========== LETTERS ==========')
+        longest_column = 0
+        for column in self.letters:
+            if longest_column < len(column):
+                longest_column = len(column)
+        
+        for letter_row in range(longest_column - 1, -1, -1):
+            for column in self.letters:
+                if letter_row < len(column):
+                    print(column[letter_row], end='')
+                else:
+                    print(' ', end='')
+            print()
+        
+        print()
+
+        print('========== GRID ==========')
+        for row in self.grid:
+            print(row)
+        
+        print()
 
-def dropquote(grid, letters):
```

Comment:
Why are we ditching the convenience method `dropquote` for using the `DropQuoteSolver` class? If you want to expose the solver object, you can just have `dropquote` return it.